source/reference/collection-level-filtering/filter-regex.txt

```diff
@@ -0,0 +1,138 @@
+.. _c2c-filter-regex:
+
+##############################
+Regular Expressions in Filters
+##############################
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: twocols
+
+.. versionadded:: 1.6
+
+.. include:: /includes/api/facts/filter-regex
+
+Syntax
+======
+
+To match databases and collections for ``mongosync`` to use :ref:`c2c-filtered-sync`
+you can use regular expressions:
+
+.. code-block:: json
+   :copyable: false
+
+   {
+      "databaseRegex": {
+         "pattern": "<string>",
+         "options": "<string>"
+      },
+      "collectionRegex": {
+         "pattern": "<string>",
+         "options": "<string>"
+      }
+   }
+
+Regular expressions in filter documents use the following fields:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Option
+     - Type
+     - Description
+
+   * - ``collectionRegex``
+     - document
+     - Specifies which collections you want the filter
+       to match.
+
+   * - ``collectionRegex.options``
+     - string
+     - Regular expression options to use in the match.
+       
+   * - ``collectionRegex.pattern``
+     - string
+     - Regular expression pattern to match.
+
+
+   * - ``databaseRegex``
+     - document
+     - Specifies which databases you want the filter 
+       to match.
+
+   * - ``databaseRegex.options``
+     - string
+     - Regular expression options to use in the match.
+
+   * - ``databaseRegex.pattern``
+     - string
+     - Regular expression pattern to match.
+
+These options are available to use with both the ``includeNamespaces``
+and ``excludeNamespaces`` parameters.
+
+Use Cases
+=========
+
+Regular expressions allow you match multiple databases or collections with a
+single pattern.  If you want to match multiple similarly named 
+databases or collections, a regular expression may be easier to match than
+creating a series of filters for individual databases or groups of collections.
+
+Details
+=======
+
+Regular Expression Options
+--------------------------
+
+``databaseRegex`` and ``collectionRegex`` each supports an ``options`` field,
+which you can use to configure regular expression options.
+
+The following options are available:
```

Comment:
These just get passed through to `$regex` under the hood. Would it be worth linking to the [$regex documentation](https://www.mongodb.com/docs/manual/reference/operator/query/regex/) somewhere in this section?

Reply:
Fixed